aws/logs_monitoring/README.md

```diff
@@ -138,6 +138,10 @@ If you can't install the Forwarder using the provided CloudFormation template, y
 
 If you encounter issues upgrading to the latest version, check the Troubleshooting section.
 
+### Upgrade an older version to +3.97.0
```

Comment:
You need to update the cloudformation template we use to deploy the forwarder to ensure we use Python 3.10 as well